server/index.ts

```
@@ -835,7 +1114,115 @@ const parseTextWithFormatting = (
         }
 
-        // Create formatted element
+        // Handle span elements with style attributes (TipTap textStyle)
+        if (tag === "span") {
+          const spanMatch = remaining.match(
+            /^(.*?)<span([^>]*)>(.*?)<\/span>(.*)$/s,
+          );
+          if (spanMatch && spanMatch[2] && spanMatch[3] !== undefined) {
+            const attributes = spanMatch[2];
+            const spanContent = spanMatch[3];
+            
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)